app/build.gradle

```diff
@@ -47,15 +46,19 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    lintOptions {
+        abortOnError false
+    }
 }
```

Comment:
Is this really needed? Builds without it, without issues or additional warnings.